### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by milktean** [alibaba/cluster-contrast-reid#29]
can I use my own dataset to train and test the model rather than using provided datasets?

**Comment by wangguangyuan** [comment]
Yes, but I suggest you to use infomap clustering, the clustering algorithm is not particularly sensitive to data distribution

**Comment by 459737087** [comment]
Hello! when I use my own dataset, but I got mAP nan%

**Comment by milktean** [comment]
> Yes, but I suggest you to use infomap clustering, the clustering algorithm is not particularly sensitive to data distribution

Thank you

**Comment by 459737087** [comment]
Hello ! Did you succeed @lkywlp99
